packages/firebase_messaging/firebase_messaging/example/lib/message.dart

```diff
@@ -31,13 +30,13 @@ class MessageView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final MessageArguments args = ModalRoute.of(context).settings.arguments;
+    final MessageArguments args = ModalRoute.of(context)?.settings.arguments as MessageArguments;
```

Comment:
```suggestion
    final MessageArguments? args = ModalRoute.of(context)?.settings.arguments;
```

Reply:
@rrousselGit : I wouldn't bother with all the stuff I did in the example app. As I said in the PR text, the only point of that was to get the official example far enough that I could use it to demonstrate that the original problem in 4949 also occurred in the example. I made no effort, to evaluate the example app changes enough to decide if they make sense or not.

The only useful thing in this PR are the two one liners in remote_message.dart. 
Everything else is only there for supporting info 

Sorry if that is confusing ...